src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -144,12 +145,37 @@ StDebugger class >> debuggerContextClass [
 	^ StDebuggerContext
 ]
 
+{ #category : #accessing }
+StDebugger class >> debuggerSettingsOn: aBuilder [
+
+	<systemsettings>
+		(aBuilder setting: #enableStackColoring)
+				label: 'Enable stack coloring';
+				target: StDebugger;
+				parent: #debugging;
+				default: true;
```

Comment:
IMO, since the color used comes from the theme, we should not override the theme. 

But one thing that would be good, would be to implement a 'configurable theme' and allow to customize it in the settings so that people can change just one color without having to implement a theme. But I don't think I'll have the tome to implement that in the near future :( Is that ok?

Reply:
No worries, it is not a priority right now.

I have though a different perspective on customizing.
Why would we block users to override the theme?

For me it is really is killing my experience when I just want to put, for example, a window in red because that window is important for me, but the system prevents me from doing that because "I should not override the system".

I agree that a good solution would be to build a powerful customizable theme where we can change things easily, but since it is not going to happen anytime soon, why not allowing that kind of small customization?

Regarding the debugger, I would like to keep that discussion open, but that will be for another time ;)